app/routes.py

```diff
@@ -1,2 +1,50 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+class Planet:
+    def __init__(self, id, name, description, num_moons):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.num_moons = num_moons
+
+    def to_dict(self):
+        return {
+                "id": self.id,
+                "name": self.name,
+                "description": self.description,
+                "num_moons": self.num_moons
+            }
+
+planets = [
+    Planet(1, "Mercury", "It's the first planet in our solar system", 0),
+    Planet(2, "Venus", "It's the second planet in our solar system", 0),
+    Planet(3, "Earth", "It's the third planet in our solar system", 1)
+]
+
+@planets_bp.route("", methods=["GET"])
+def handle_planets():
+    planets_response = []
+    for planet in planets:
+        planets_response.append(planet.to_dict())
```

Comment:
This would be a good candidate for a list comprehension, what could that look like?